static/css/components/work.less

```diff
@@ -24,8 +24,14 @@ div.editionAbout {
     color: @dark-grey;
     margin: 0 0 10px;
   }
+
+  h3 {
+    font-size: 14px;
```

Comment:
not thrilled about this solution, but let's go with it as a stop-gap until we can get reasonable looking font-sizes on the books page

Reply:
How does an h3 impact a button? Bit confused about scope of this patchset.